CRM/Export/BAO/Export.php

```diff
@@ -1242,11 +1101,6 @@ public static function exportComponents(
         self::mergeSameHousehold($exportTempTable, $headerRows, $sqlColumns, $relationKeyHOH);
       }
 
-      // fix the headers for rows with relationship type
```

Comment:
Should we do this once per row .. or perhaps just once per export. Let's go for the latter